compiler/il/OMRDataTypes.hpp

```diff
@@ -349,18 +349,20 @@ namespace TR
 namespace OMR
 {
 
+#define OMR_CREATE_VECTOR_TYPE(elementType, length) (static_cast<TR::DataTypes>(TR::NumScalarTypes + (length - 1) * TR::NumVectorElementTypes + elementType - 1))
```

Comment:
I imagine that callers should still use `createVectorType()` instead of this whenever possible, i.e. whenever the result doesn't need to be a compile-time constant. If so, I think this should have a comment to that effect, and maybe a more obnoxious name

Reply:
Done